tensorboard/uploader/uploader_main.py

```diff
@@ -538,6 +562,23 @@ def _die(message):
   sys.exit(1)
 
 
+def _format_time(timestamp_pb):
+  # Add and subtract a day for <https://bugs.python.org/issue29097>,
+  # which breaks early datetime conversions on Windows for small
+  # timestamps.
+  dt = datetime.datetime.fromtimestamp(timestamp_pb.seconds + 86400)
+  dt = dt - datetime.timedelta(seconds=86400)
+
+  ago = datetime.datetime.now().replace(microsecond=0) - dt
+  if ago < datetime.timedelta(seconds=5):
+    return "just now"
```

Comment:
How would you feel about providing the exact timestamp in all cases and maybe just add the human-friendly interpretations of recent times in parentheses after the timestamp or something?  Maybe just my pet peeve, but as a user I'm often a little annoyed when there's a humanized timestamp with no way to access the exact value (e.g. via the title text or something).

Reply:
Totally agree; uncharacteristic of me to omit that. Will change.